src/tests/helper_tests.cxx

```diff
@@ -0,0 +1,226 @@
+  std::vector<char> expected = {};
+  auto actual = gulp("1");
+  ASSERT_EQ(expected, actual);
+}
+
+TEST(HelperTests, gulp_returns_correct_binary_when_file_is_read) {
+  std::fstream fs("gulptest.txt", std::ios::out | std::ios::binary);
+  fs.write("hello", 5);
+  fs.close();
+  std::vector<char> expected = {'h', 'e', 'l', 'l', 'o'};
+  auto actual = gulp("./gulptest.txt");
+  ASSERT_EQ(expected, actual);
+}
+
+TEST(HelperTests, gulp_returns_empty_vector_when_empty_file_is_given) {
+  std::vector<char> expected = {};
+  auto actual = gulp("");
+  ASSERT_EQ(expected, actual);
+}
+
+TEST(HelperTests, get_string_returns_exit_command) {
+  std::string expected = "exit";
+  rapidjson::Document j0;
+  j0.Parse("{\"cmd\": \"exit\"}");
+
+  std::string actual = get_string(&j0, "cmd");
+
+  ASSERT_EQ(expected, actual);
+}
+
+TEST(HelperTests, get_string_returns_add_command_with_multiple_streams) {
+  std::string expected = "add";
+  rapidjson::Document j0;
+  j0.Parse("{\n"
+           "  \"cmd\": \"add\",\n"
+           "  \"streams\": [\n"
+           "    {\n"
+           "      \"channel\": \"<EPICS PV name>\",\n"
+           "      \"converter\": {\n"
+           "        \"schema\": \"<schema-id>\",\n"
+           "        \"topic\": \"<Kafka-topic>\"\n"
+           "      }\n"
+           "    },\n"
+           "    {\n"
+           "      \"channel\": \"<EPICS PV name..>\",\n"
+           "      \"converter\": {\n"
+           "        \"schema\": \"<schema-id>\",\n"
+           "        \"topic\": "
+           "\"//<host-if-we-do-not-like-the-default-host>[:port]/"
+           "<Kafka-topic..>\"\n"
+           "      }\n"
+           "    },\n"
+           "    {\n"
+           "      \"channel\": \"<EPICS Channel Access channel name>\",\n"
+           "      \"channel_provider_type\": \"ca\",\n"
+           "      \"converter\": {\n"
+           "        \"schema\": \"<schema-id>\",\n"
+           "        \"topic\": \"<Kafka-topic..>\"\n"
+           "      }\n"
+           "    }\n"
+           "  ]\n"
+           "}");
+
+  std::string actual = get_string(&j0, "cmd");
+
+  ASSERT_EQ(expected, actual);
+}
+
+TEST(HelperTests, get_string_returns_number_given_number_in_array) {
+  std::string expected = "2";
+  rapidjson::Document j0;
+  j0.Parse("[\"1\", \"2\", \"3\"]");
+
+  std::string actual = get_string(&j0, "1");
+
+  ASSERT_EQ(expected, actual);
+}
+
+TEST(HelperTests, get_string_returns_empty_string_if_array_is_empty) {
+  std::string expected;
+  rapidjson::Document j0;
+  j0.Parse("[]");
+  std::string actual = get_string(&j0, "1");
+
+  ASSERT_EQ(expected, actual);
+}
+
+TEST(HelperTests, get_string_returns_empty_string_if_empty_json) {
+  std::string expected;
+  rapidjson::Document j0;
+  j0.Parse("");
+  std::string actual = get_string(&j0, "1");
+
+  ASSERT_EQ(expected, actual);
+}
+
+TEST(HelperTests, split_with_empty_string_returns_empty_vector) {
+  using std::vector;
```

Comment:
Don't use `using std::vector` and `using std::string` for these tests, it is just adding lines.